netci.groovy

```diff
@@ -797,7 +797,9 @@ combinedScenarios.each { scenario ->
                                 case 'arm64':
                                     assert scenario == 'default'
                                     buildCommands += "build.cmd ${lowerConfiguration} ${architecture} skiptestbuild /toolset_dir C:\\ats"
-                                    // Add archival.  No xunit results for x64 windows
+                                    buildCommands += "C:\\arm64PostBuild.cmd ${WORKSPACE} ${architecture} ${lowerConfiguration}"
```

Comment:
I think you actually want %WORKSPACE% here.  The job that is generating these jobs has a workspace itself.  That's what ${WORKSPACE} will evaluate to, rather than what you want, which is the WORSPACE variable for the arm64 job itself